config.yaml

```diff
@@ -1,49 +1,62 @@
 options:
   path:
-      default: ""
       type: string
-      description: "Either an organization name or repository in 'org/repo' format"
-  token:
       default: ""
+      description: >
+        The path of the GitHub repo to register the self-hosted runners. Either an organization name
+        or repository in 'org/repo' format.
+  token:
       type: string
-      description: "Personal Access Token, required for auto registerting with the given url"
-  quantity:
-      default: 3
-      type: int
-      description: |
-        DEPRECATED: use `containers` and `virtual-machines`
-        Quantity of runners to maintain
+      default: ""
+      description:
+        The GitHub Personal Access Token for auto-registering the self-hosted runners.
```

Comment:
"auto-registering" -> "registering"? or maybe "for the self-hosted runners to self register"?

Reply:
fixed.